apps/artist/client/views/overview.coffee

```diff
@@ -71,21 +71,13 @@ module.exports = class OverviewView extends Backbone.View
     return if not @statuses.shows
     $el = @$('.artist-overview-header .artist-exhibition-highlights')
     # If there are more than 15 shows, take ten and show a 'see more' link
-    # If there are less than 15 shows, show them all.
+    # If there are less than 15 shows, show them all, grouped by kind of show.
     showMore = counts.shows > 15
     if showMore
       highlights = viewHelpers.nShowsByDate(shows, 10)
     else
-      solo = []
-      group = []
-      fair = []
-      shows = _.each viewHelpers.nShowsByDate(shows, 15), (show) ->
-        if show.fair
-          fair.push show
-        else if show.artists.length > 1
-          group.push show
-        else
-          solo.push show
+      { solo, group, fair } = _.groupBy(viewHelpers.nShowsByDate(shows, 15), 'kind')
```

Comment:
#no-op and this way is better b/c Metaphysics already has advanced logic for `kind`: https://github.com/artsy/metaphysics/blob/da3181da9cf6213786d731d902b3d038f802782d/schema/show.js#L61